bpf/bpf_lxc.c

```diff
@@ -113,7 +113,7 @@ static __always_inline int __per_packet_lb_svc_xlate_4(void *ctx, struct iphdr *
 	/* Store state to be picked up on the continuation tail call. */
 	lb4_ctx_store_state(ctx, &ct_state_new, proxy_port);
 	ep_tail_call(ctx, CILIUM_CALL_IPV4_CT_EGRESS);
-	return ret;
```

Comment:
That looks like a bug to and should be marked as such and backported. It means we fail to report some missed tail calls AFAIU and probably pass the packet to the stack instead.

Reply:
Discussed offline - This is a bug, but it hasn't made it to any release yet, so we just want to fix this on `master`.

Reply:
And it's also without consequence until another bug causes us to fail the tail call, so all good.